src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -782,7 +782,7 @@ internal async Task<Collection<FunctionDescriptor>> GetFunctionDescriptorsAsync(
             return functionDescriptors;
         }
 
-        internal static void ValidateFunction(FunctionDescriptor function, Dictionary<string, HttpTriggerAttribute> httpFunctions)
+        internal static void ValidateFunction(FunctionDescriptor function, Dictionary<string, HttpTriggerAttribute> httpFunctions, IEnvironment environment = null)
```

Comment:
I don't think environment should be optional, since it's needed in all cases to do proper validation

Reply:
Fixed